LondonBritishLibrary/orient/BLorient589.xml

```diff
@@ -54,49 +53,64 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                                 </support>
                                 <extent>
                                     <measure unit="leaf">98</measure>
-                                    <dimensions type="outer" unit="mm">
-                                        <height>165</height>
-                                        <width>121</width>
+                                    <measure unit="leaf" type="blank">3</measure>
+
+                                    <dimensions type="outer" unit="in">
+                                        <height>6.5</height>
+                                        <width>4.75</width>
                                     </dimensions>
                                 </extent>
-                                
-                                
+                                      <foliation>Two unnumbered blank leaves after <locus target="#1r">f. 1r</locus> and one at the end.</foliation>
                             </supportDesc>
-                            
+
                             <layoutDesc>
                                 <layout columns=" 2" writtenLines="21" cert="high"/>
                             </layoutDesc>
                         </objectDesc>
-                        
+
                         <handDesc>
-                            
+
                             <handNote script="Ethiopic" xml:id="h1">
                                 <desc>The manuscript is neatly written of the <date>18th century</date>
                                 </desc>
-                                
+
                                 <date notBefore="1700" notAfter="1800"/>
-                                
+
                             </handNote>
-                            
-                        </handDesc>   
+
+                        </handDesc>
                         <additions>
-                            
+                          <list>
+                          <item xml:id="a1">
+                              <locus target="#2r"/>
+                              <desc type="OwnershipNote">The note states that the manuscript belongs to the church of <placeName ref="INS0101MadhaneAlam">Madḫāne ʿAlam</placeName>
+                              </desc>
+                              <q xml:lang="gez">ምዕራፍ፡ ዘቅ' ፡ መ' ፡ ዓለም።</q>
```

Comment:
ዘቅ' ፡ መ' ፡ I think this is an abrreviation by Wright. Could you please, either write it out in full or encode the omission?